automatic/1password7/1password7.nuspec

```
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>1password7</id>
+    <version>0.0</version>
+    <title>1Password</title>
+    <authors>AgileBits</authors>
+    <owners>chocolatey-community, AgileBits</owners>
+    <licenseUrl>https://1password.com/legal/terms-of-service</licenseUrl>
+    <projectUrl>https://1password.com/</projectUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@09451a71b28e4ee0b3ea3841ab130b1bbf46f9b0/icons/1password7.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <mailingListUrl>https://discussions.agilebits.com/</mailingListUrl>
+    <!-- Do not touch the description here in the nuspec file. Description is imported during update from the Readme.md file -->
+    <description><![CDATA[]]></description>
+    <summary>1Password - Have you ever forgotten a password?</summary>
+    <copyright>©2021 AgileBits, Inc. All rights reserved.</copyright>
+    <tags>1password password keystore keys saver admin trial cross-platform</tags>
```

```suggestion
    <tags>1password7 productivity trial cross-platform password keystore keys</tags>
```
We could possibly update the tags to [the guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md#obligatory-tags-and-categories)

---

I'm not sure if `productivity` is the correct one, maybe just use the `utility` tag instead?

---

I used utility for the tags instead of `productivity`.